src/Hint/Context.hs

```diff
@@ -128,7 +137,11 @@ addPhantomModule mod_text =
                    (old_top, old_imps) <- runGhc getContext
                    --
                    runGhc $ GHC.addTarget t
+#if MIN_VERSION_ghc(9,4,0)
+                   res <- runGhc $ GHC.load GHC.LoadAllTargets
+#else
                    res <- runGhc $ GHC.load (GHC.LoadUpTo m)
+#endif
```

Comment:
Couldn't we use `LoadAllTargets` for all GHC versions now?

Reply:
Perhaps... I guess I can try on the oldest supported version.